pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -491,7 +508,16 @@ def _update_doc_in_db(
         tes_uri: str,
         remote_task_id: str,
     ) -> DbDocument:
-        """Update the document in the database."""
+        """Update the document in the database.
+
+        Args:
+            db_connector: The database connector.
+            tes_uri: The tes uri where the task if forwarded.
```

Comment:
Please capitalize both "TES" and "URI" (here and in all other docstrings).

Reply:
Done